src/metrics.rs

```diff
@@ -2415,6 +2434,9 @@ impl GooseAttack {
             if self.configuration.status_codes {
                 merge_request.set_status_code(request_metric.status_code);
             }
+            if !self.configuration.report_file.is_empty() {
+                merge_request.record_requests_per_second(self.metrics.duration);
```

Comment:
We already have `self`, so no need to pass `self.metrics.duration`

Reply:
The graph should include ramp-up time and ramp-down time -- in the following test I ran with the following parameters:

```bash
cargo run --release --example drupal_memcache -- --host http://apache/ -v -u5 -t10 --report-file report.html --throttle-requests 11
```

It's not including the ramp-up time, and it needs to. Also, I think we're missing the first second, as we should see 10 seconds of requests for ~11 rps:

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/402892/141246254-b88e9eb9-dedf-46ae-adf4-312d5bf4b620.png">

Reply:
> We already have self, so no need to pass self.metrics.duration

`record_requests_per_second()` is a function on a different struct so `self` represents  something else there. And `duration` is not available there, so we need to send it.